lib/galaxy/webapps/galaxy/api/users.py

```diff
@@ -229,7 +229,6 @@ def update(self, trans, id, payload, **kwd):
         self.user_deserializer.deserialize(user_to_update, payload, user=current_user, trans=trans)
         return self.user_serializer.serialize_to_view(user_to_update, view='detailed')
 
-    @web.require_admin
```

Comment:
> @dannon we should also talk security, since this is no longer protected by admin status

@astrovsky01 Yes, definitely.  You'll need to check inside the method if the individual making the request is an admin or if the target matches.  As it is, just dropping @admin would allow anyone to delete arbitrary users.

Reply:
I have the secondary request for inputting of the user's email and check it against who is logged in, but I feel as though password input should also be necessary.

Reply:
I did see that, but that's not enough as it's purely clientside.  Never trust the client :)

One could skip the interface and just do:

`curl -X "DELETE" http://localhost:8080/api/users/NOTMYUSER`

And it'd work.  This must have serverside validation -- you'll want to compare `(trans.user_is_admin or trans.user == user_to_delete)`

Reply:
I'd figured, yeah.